ui/shared/RecipientSelector.qml

```diff
@@ -18,7 +18,21 @@ Item {
     height: (readOnly ? inpReadOnly.height : inpAddress.height) + txtLabel.height
     //% "Invalid ethereum address"
     readonly property string addressValidationError: qsTrId("invalid-ethereum-address")
+    property bool isValid: false
+    property var reset: function() {}
+    readonly property var sources: [qsTr("Address"), qsTr("Contact"), qsTr("My account")]
```

Comment:
Can you put those on individual lines for the translation script to work, thanks

Reply:
Updated.